packages/screeb-sdk-browser/docs/README.md

```diff
@@ -354,6 +354,8 @@ This is the Screeb tag options object.
 | Name | Type | Description |
 | :------ | :------ | :------ |
 | `screebEndpoint?` | `string` | Please don't do this. |
+| `sdkName?` | `string` | SDK name (eg: sdk-browser, sdk-react, sdk-angular, etc...) |
```

Comment:
pas sûr qu'on veuille laisser le dev remplir ce truc

WDYT ?

sauf s'il faisait lui-même un wrapper communautaire ?

Reply:
Pas faux, pour le coup c'est généré mais je peux simplement le cacher dans la doc, as you wish !

Reply:
comme tu veux